appveyor.yml

```diff
@@ -20,6 +20,11 @@ notifications:
     on_build_status_changed: false
 
 install:
+# If there's a newer build queued for the same PR, cancel this one
```

Comment:
I see you enabled rolling builds. That's the appveyor built-in version of this that actually does a pre-emptive cancel as opposed to needing the VM to start up first, but note that it's not always what you want as if there are many commits to master, you may actually want all of them to run.